Makefile

```diff
@@ -79,19 +79,27 @@ debug: build
 #   GOPROXY:   it specifies the download address of the dependent package
 #   REGION:    region to build. e.g.: 'cn' means china mainland(the default value),
 #              the script will set the remote mirror address according to the region to speed up the build.
+#   DOCKER_BUILD_ARGS: additional parameters of the Dockerfile need to be passed in during the image building process,
+#                      e.g.: `--build-arg BUILD_FLAGS=-race`
+#   DOCKERFILE_PATH:   the Dockerfile path used by docker build, default is `kubeadmiral/hack/dockerfiles/Dockerfile`
 #
 # Examples:
 #   # build images with the host arch, it will generate "ghcr.io/kubewharf/kubeadmiral-controller-manager:latest"
 #   make images
 #
-#   # build images with amd64,arm64 arch
+#   # build images with amd64,arm64 arch, and use the mirror source in mainland China during the construction process
 #   # note: If you specify multiple architectures, the image tag will be added with the architecture name,e.g.:
 #   # 		"ghcr.io/kubewharf/kubeadmiral-controller-manager:latest-amd64"
 #   # 		"ghcr.io/kubewharf/kubeadmiral-controller-manager:latest-arm64"
-#   make images ARCHS=amd64,arm64
+#   make images ARCHS=amd64,arm64  REGION="cn"
+#
+#   # pass parameters to go build and enter debug mode
```

Comment:
```suggestion
#   # enable race detection in go build
```

Reply:
the note has been modified, please take a review.